src/constants.ts

```diff
@@ -93,6 +93,7 @@ export const TWO = JSBI.BigInt(2)
 export const THREE = JSBI.BigInt(3)
 export const FIVE = JSBI.BigInt(5)
 export const TEN = JSBI.BigInt(10)
+export const SECONDS_IN_YEAR = JSBI.BigInt(31557600)
```

Comment:
I wonder if we should use 365 days instead of 365.25 . . . I would think the average person would default to thinking of a year as 365 days.

Reply:
Gotcha, updated here: cc5286da0da70a6bad1277be8231afcb50a44757